composer.json

```diff
@@ -116,7 +116,8 @@
                 "Integrity constraint violation: 1048 Column 'langcode' cannot be null": "https://www.drupal.org/files/issues/2019-07-10/paragraphs-set_langcode_widgets-2901390-29.patch"
             },
             "drupal/private_message": {
-                "3248258 - Coding standard issue at PrivateMessageService": "https://www.drupal.org/files/issues/2021-11-08/coding-standard-3248258-2.patch"
+                "3248258 - Coding standard issue at PrivateMessageService": "https://www.drupal.org/files/issues/2021-11-08/coding-standard-3248258-2.patch",
+                "Move message notify to a submodule so it doesn't crashes on update": "https://git.drupalcode.org/project/private_message/-/merge_requests/31.diff"
```

Comment:
Hi Robert, Ronald made a comment to a [similar situation here](https://github.com/goalgorilla/open_social/pull/2748/files#r802637677) we should follow the same approach :D